tests/plugins/org.polarsys.capella.test.rcptt/DiagramNavigation/DiagramNavigationOnDoubleClickFalseTest.test

```diff
@@ -0,0 +1,243 @@
+--- RCPTT testcase ---
```

Comment:
general comment, can you avoid to write rcptts as most as possible, as they are not launched on PR and not easily maintainable.
this kind of tests can be written in junit by retrieving the view, setting the selection and launch doubleclick programatically for instance